README.md

```diff
@@ -1529,6 +1549,63 @@ $ ./bin/neo-go contract invokefunction -r http://localhost:20331 -w my_wallet.js
 $ ./bin/neo-go contract invokefunction -r http://localhost:20331 -w my_wallet.json 9042814f07d65d2b835fa1f07d21c22c6e1cbdf7 delete my_second_domain -- NbrUYaZgyhSkNoRo9ugRyEMdUZxrhkNaWB
 ```
 
+## Workshop. Part 5
+
+In this part we'll take a look at the example of a decentralized application for the
+Neo ecosystem written in Go. The example demonstrates how to use NeoGo RPC Client
+and a various set of helpers to work with wallets, deploy and invoke smart
```

Comment:
and a set
work with wallets, deployments, invocations, data retrieval and such

Reply:
Fixed.